examples/roles.yml

```diff
@@ -1,2 +1,2 @@
 - src: azavea.pip
-  version: 0.1.1
+  version: 1.0.0
```

Comment:
With the removal of the dependencies required to support SNI, I think that the `pip` role can be removed too.